src/Microsoft.Tye.Hosting/TyeHost.cs

```diff
@@ -298,6 +313,32 @@ private async Task StopAsync()
             _processor = null;
         }
 
+        private static void OpenDashboard(string url)
+        {
+            try
+            {
+
+                //still not addressed https://github.com/dotnet/corefx/issues/10361
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                {
+                    url = url.Replace("&", "^&");
+                    System.Diagnostics.Process.Start(new ProcessStartInfo("cmd", $"/c start {url}") { CreateNoWindow = true });
+                }
+                else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+                {
+                    System.Diagnostics.Process.Start("xdg-open", url);
+                }
+                else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+                {
+                    System.Diagnostics.Process.Start("open", url);
+                }
+            }
+            catch
+            {
+                throw;
```

Comment:
Why throw here instead of logging directly?

Reply:
method was static, _logger not available will make the change